public/index.html

```diff
@@ -154,6 +155,19 @@
     <app-box class="first" effects="parallax-background"></app-box>
   </section>
 
+  <!-- Tournament -->
+  <section>
```

Comment:
I wouldn't worry about adding it to the front page just yet. We could just have the single tournament at https://www.hundredzeros.net/zeromania.